projects/v2/src/app/modules/navbar/navbar.component.ts

```diff
@@ -84,6 +96,11 @@ export class NavbarComponent implements OnInit {
         this.SHEET_OPTIONS = SHEET_OPTIONS;
       }
     });
+
+    this.selectedOrgans$.subscribe((organs) => {
+      this.selectedOrgans = organs;
+      this.selectedOrgansValues = organs?.join(', ');
```

Comment:
Need to work on making this cleaner on the UI. Play with this a bit, but I'm thinking comma separated organ names (Ie ''Heart, Brain, ...") if the string ends up being less than 64 characters. Otherwise "n Organs Selected").